draft-ietf-quic-http.md

```diff
@@ -334,6 +334,10 @@ them.  However, stream types which could modify the state or semantics of
 existing protocol components, including QPACK or other extensions, MUST NOT be
 sent until the peer is known to support them.
 
+A sender can close or reset a unidirectional stream unless specified otherwise.
```

Comment:
```suggestion
A sender can close or reset a unidirectional stream unless otherwise specified.
```
This is the more usual word ordering, though I admit I don't know why.  :-)

Reply:
https://books.google.com/ngrams/graph?content=unless+specified+otherwise%2C+unless+otherwise+specified&year_start=1800&year_end=2000&corpus=15&smoothing=3&share=&direct_url=t1%3B%2Cunless%20specified%20otherwise%3B%2Cc0%3B.t1%3B%2Cunless%20otherwise%20specified%3B%2Cc0

Reply:
Interesting data, but it doesn't argue strongly either direction.  One is vastly more common but having a decline, while the other is uncommon but gaining.

Reply:
Thank you for the suggestion and the discussion. I've applied the suggested fix assuming that @LPardue would not complain "unless otherwise specified" to be incorrect.